e2e/suites/actions/library-actions.test.ts

```diff
@@ -89,12 +89,9 @@ describe('Library actions', () => {
     await adminApiActions.sites.createSite(siteSearchModerated2Admin, SITE_VISIBILITY.MODERATED);
     await apis.user.sites.createSite(siteSearchForDelete);
 
-    await apis.user.queries.waitForSites('site-public-search', { expect: 5 });
-    await apis.user.queries.waitForSites('site-moderated-search', { expect: 2 });
-
     await loginPage.loginWith(username);
     done();
-  });
+  }, 300000);
```

Comment:
there must be another way instead of using this crazy big timeout

Reply:
5 minutes is extreme?

Reply:
for creating 7 sites yes, 5 mins is a lot